fv3core/stencils/dyn_core.py

```diff
@@ -291,6 +291,17 @@ def __init__(
             origin=self.grid.full_origin(),
             domain=self.grid.domain_shape_full(),
         )
+        ax_offsets_pe = axis_offsets(
```

Comment:
Could this be refactored into a function so that the code is not duplicated in the translate class?